Description: Do not depends on argparse in Python 2.7
Author: Julien Danjou <acid@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: other
Forwarded: https://review.openstack.org/2464
Last-Update: 2011-12-19

--- python-novaclient-2012.1~e2.orig/setup.py
+++ python-novaclient-2012.1~e2/setup.py
@@ -17,9 +17,11 @@ import setuptools
 import sys
 
 
-requirements = ["httplib2", "argparse", "prettytable"]
+requirements = ["httplib2", "prettytable"]
 if sys.version_info < (2, 6):
     requirements.append("simplejson")
+if sys.version_info < (2, 7):
+    requirements.append("argparse")
 
 
 def read_file(file_name):
